src/components/createScanDialog/createScanDialog.js

```diff
@@ -343,6 +345,7 @@ class CreateScanDialog extends React.Component {
           <TextArea
             isDisabled={!checked.jbossEap && !checked.jbossFuse && !checked.jbossWs && !checked.jbossBrms}
             name="displayScanDirectories"
+            ouiaId="display_scan_directories"
```

Comment:
required: Apply the same automagic of `name` and autogenerated `ouiaId` for `TextInput` to `TextArea`

Reply:
Good catch! I think as long as your testing framework can read it `naive` works. It's a functional update that can be expanded or relaxed if/when the next version of Patternfly is implemented